Pyjion/absint.cpp

```diff
@@ -212,24 +238,23 @@ bool AbstractInterpreter::interpret() {
         int oparg;
         auto cur = queue.front();
         queue.pop_front();
-        for (size_t curByte = cur; curByte < m_size; curByte++) {
+        for (size_t curByte = cur; curByte < m_size; curByte += sizeof(_Py_CODEUNIT)) {
             // get our starting state when we entered this opcode
             InterpreterState lastState = m_startStates.find(curByte)->second;
 
             auto opcodeIndex = curByte;
 
-            auto opcode = m_byteCode[curByte];
-            if (HAS_ARG(opcode)) {
-                oparg = NEXTARG();
-            }
+            auto opcode = GET_OPCODE(curByte);
+			oparg = GET_OPARG(curByte);
 
         processOpCode:
```

Comment:
Yep, just different functions (preprocess, interpret, and it occurs again in compile_worker)